### DISCUSSION TITLE
Add support for `pylance.analysis.exclude` setting

### DISCUSSION THREAD

**Discussion by zbs** [microsoft/pylance-release#2581] [Enhancement]
To prevent pylance from indexing particularly large subdirectories (a la `node_modules`), or symlinks to such, it would be nice to have a VS Code setting (e.g. `pylance.analysis.excludes`) that mimics [the exclude setting from `pyrightconfig.json`](https://github.com/microsoft/pyright/blob/master/docs/configuration.md#main-pyright-config-options) instead of having to manage a pyrightconfig.json file in your git repo.

@jakebailey @erictraut

**Comment by judej** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.

**Comment by luabud** [comment]
This is now available on version  2022.9.11. You can check our [changelog](https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#2022911-7-september-2022-prerelease) to learn more.
